### PR TITLE
Update landing page meta descriptions

### PR DESCRIPTION
**Pull request by kewitham**
This PR updates the meta descriptions per [this issue](https://github.com/usdoj-crt/beta.ADA.gov-Project-Management/issues/830).

### CODE REVIEW

File: _pages/file-a-complaint.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: File a Complaint
-description: "The Americans with Disabilities Act provides an important tool to fight discrimination: filing a complaint with an appropriate federal agency.  This page outlines the steps to get you started."
+description: "The ADA provides an important tool to fight discrimination: filing a complaint with an appropriate federal agency. This page outlines the steps to do so."
```
Comment (okkays) [inline _pages/file-a-complaint.md:3]: supernit: me being [foolishly consistent](https://peps.python.org/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds)

```suggestion
description: The ADA provides an important tool to fight discrimination: filing a complaint with an appropriate federal agency. This page outlines the steps to do so.
```

**Review by okkays — APPROVED** [review]
LGTM!

File: _pages/file-a-complaint.md (hunk 1 above)
Comment (kewitham) [reply _pages/file-a-complaint.md:3]: I added the quotes because otherwise the ':' breaks the field

Comment (okkays) [reply _pages/file-a-complaint.md:3]: ah, nice, makes sense

**Review by vwilliamsn — APPROVED** [review]
LGTM!

**Pull request merged**